tests/system/all_sets.cfg

```diff
@@ -168,11 +168,51 @@ print_statements  = True
 
 [#]
 sets = ["enso_diags"]
-plot_type = "scatter"
-case_id = "TREFHT"
+case_id = 'TREFHT-response-map-start-yrs'
 debug = True
+diff_colormap = "BrBG"
+output_format_subplot = ["png"]
+regions = ["20S20N"]
 results_dir = "all_sets_results_test"
+seasons = ["ANN"]
+test_start_yr = '2012'
```

Comment:
@chengzhuzhang Keeping this test, as it checks that we can mix and match the different `{start, end}_yr` parameters.